### PR TITLE
[syncd] Process all cases fdb flush notification

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: syncd/syncd_notifications.cpp [hunk 1]
```diff
@@ -176,13 +173,34 @@ void redisPutFdbEntryToAsicView(
...
+
+                    if (bp && *bp == strBridgePortId)
+                    {
+                        SWSS_LOG_DEBUG("remove fdb entry %s for SAI_FDB_EVENT_FLUSHED",fdbkey.c_str());
+                        g_redisClient->del(fdbkey);
+                    }
+                }
+                else
+                {
+                    SWSS_LOG_ERROR("found unknown type fdb entry, key %s", fdbkey.c_str());
```
Comment (prsunny) [inline syncd/syncd_notifications.cpp:198]: With FLUSH_TYPE ALL,  should we also get flush notification for static?

```
    /** Flush static and dynamic FDB entries */
    SAI_FDB_FLUSH_ENTRY_TYPE_ALL,
```

Comment (kcudnik) [reply syncd/syncd_notifications.cpp:198]: yes

Comment (kcudnik) [reply syncd/syncd_notifications.cpp:198]: actually

Comment (kcudnik) [reply syncd/syncd_notifications.cpp:198]: no

Comment (kcudnik) [reply syncd/syncd_notifications.cpp:198]: this is notification processing, and notification don't get type type, that's why if vendor wants to flush all static and non static, then its required for vendor to send 2 flush notifications, one, when type is set to dynamic, and second for static

Comment (kcudnik) [reply syncd/syncd_notifications.cpp:198]: since sai_fdb_event_notification_data_t contains sai_fdb_event_t event_type, and not sai_fdb_flush_entry_type_t

Comment (kcudnik) [reply syncd/syncd_notifications.cpp:207]: fixed

**Review by prsunny — APPROVED** [review]

**Pull request merged**